sdk/react/widgets/VideoViewPlayPause.js

```diff
@@ -102,22 +115,22 @@ class VideoViewPlayPause extends React.Component {
 
   onAnimationCompleted = (instance) => {
     this.state.widget.animationOpacity.setValue(this.props.showButton ? 1 : 0);
-    this.setState({inAnimation:false});
+    this.setState({inAnimation: false});
     this.syncButtons(this.props.playing);
   };
 
   // Animations for play/pause transition
   animatePlayButton = () => {
     this.setState({inAnimation: true});
-    this.state.play.animationScale.setValue(1);
```

Comment:
I'm slightly confused about these changes. You are reversing the animation to go from invisible to visible instead of the other way around. But why?

Reply:
@pilievOoyala before animation was wrong. It started from size x1 and animated to size x2. After animation's end function `showPlayButton` was called, where we changed scale from value `x2` to value `x1`. And on this behavior we had a non-animated play button size changes. It looked a bit strange.